controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ContactInformationMaintainer.java

```diff
@@ -29,28 +36,60 @@
     private static final Logger log = Logger.getLogger(ContactInformationMaintainer.class.getName());
 
     private final Organization organization;
+    private final String apiName;
```

Comment:
apiName - is actually hostname? And it is coming from config?

To make the intention clear as you comment below maybe we should take in a  list of baseUrls for systems to be feed to?